src/main/java/frc/robot/commands/intake/IntakePistonCmd.java

```diff
@@ -0,0 +1,53 @@
+/*----------------------------------------------------------------------------*/
```

Comment:
Each of these command files is 99% boilerplate and 1 line of useful code.  These should all be converted to lambdas in configureButtonBindings.